components/UserList.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import Player from './Player'
+
+/**
+ * List of players component
+ */
+export default function UserList(props) {
+  const displayPlayers = props.userList.map((player) => (
+    <li className="w-full" key={player.id}>
+      {props.currPlayer.id === player.id ? (
+        // Current player.
+        <Player
+          playerName={player.playerName}
+          playerCard={player.playerCard}
+          selectedCard={props.selectedCard}
+          key={props.selectedCard}
+        />
+      ) : (
+        // Other players.
+        <Player
+          playerName={player.playerName}
+          playerCard={player.playerCard}
+          key={props.selectedCard}
+        />
+      )}
+    </li>
+  ))
+  return (
+    <div className="rounded border p-2 flex flex-col bg-white">
+      <div className="flex justify-between border-b-2 border-slate-300 p-2 text-lg font-display font-semibold text-slate-700 tracking-wide">
+        <p className="px-2">{props.t.users}</p>
+        <p className="px-2">{props.t.card}</p>
+      </div>
+      <ul className="flex flex-col " data-testid="cardList">
+        {props.userList ? displayPlayers : ''}
+      </ul>
+      {props.children}
+    </div>
+  )
+}
+
+UserList.propTypes = {
```

Comment:
Some propTypes missing:
`t` for the translated text,
`selectedCard` for the current selected card
`currPlayer` for the current player in the session

Reply:
fixed